### DISCUSSION TITLE
为什么说「可视化设计流程图」不可行呢

### DISCUSSION THREAD

**Discussion by lili21** [taowen/awesome-lowcode#52] [General]
> 【今日阅读】bret victor 的经典之作 learnable programming（http://worrydream.com/LearnableProgramming/），中文译本 http://pythonic.zoomquiet.top/data/20150419235328/index.html 。推荐语：可视化设计流程图在《没有银弹》成文的1986年代已经是被吊起来打的批判对象了，“现代的”low code 平台还在拿可视化设计流程图来做为卖点不能不说是可悲的。bret victor 的这篇经典文章用大量的例子可视化的说明了啥叫真正的可视化。

**Comment by LowCoderGitHub** [comment]
搞了个低代码行业的交流平台，大佬有没有兴趣来新平台玩玩
https://www.lowcoder.com.cn/
